test/framework/deployment_helpers_test.go

```diff
@@ -68,7 +68,7 @@ controller_runtime_reconcile_panics_total{controller="clusterclass"} 0
 # TYPE controller_runtime_webhook_panics_total counter
 controller_runtime_webhook_panics_total 0
 `),
-			wantErr: "panic occurred in \"cluster\" controller",
+			wantErr: "1 panics occurred in \"cluster\" controller (check logs for more details)",
```

Comment:
Are these tests not executed in CI?

They were not green before this change, right?